backend/core/src/main/java/org/sonarsource/sonarlint/core/analysis/AnalysisService.java

```diff
@@ -498,15 +474,19 @@ public void onStandaloneRulesConfigurationChanged(StandaloneRulesConfigurationCh
     }
   }
 
-  public UUID analyzeOpenFiles(String configScopeId) {
+  @CheckForNull
```

Comment:
RPC client is not one of the callers now, it's the single caller, so we might want to review what's going on here.

Reply:
It is. `forceAnalyzeOpenFiles` is called by `org.sonarsource.sonarlint.core.rpc.impl.AnalysisRpcServiceDelegate#analyzeOpenFiles`